src/cpp/r/RCntxt.cpp

```diff
@@ -102,7 +102,7 @@ bool RCntxt::isDebugHidden() const
 bool RCntxt::isErrorHandler() const
 {
    SEXP errFlag = r::sexp::getAttrib(callfun(), "errorHandlerType");
-   return TYPEOF(errFlag) == INTSXP;
+   return errFlag != R_NilValue;
```

Comment:
Note that these types were changed from integer to character in https://github.com/rstudio/rstudio/commit/9e3d350b25fe4ea3165e742b25c5f71cfa1bf9c2#diff-fcd443acd6f8b9a49ae244cf34eb175ffc1cc65f93f3c6efba80b452397bebf5.